### PR TITLE
Created PRS in BLorient2082 (No.1)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
All of them are mentioned in a colophon at the end of a synaxary. I hope, that I identified them correctly as names. For some of these names, incertainties remained on my side: 

1) PRS14454EhataKrestos could possibly be the prominent figure, that was venerated as a saint later. 
2) PRS14457WaldaHawWS could be two persons. Either, I do not know, whether ምስክር is a title to PRS14457WaldaHawWS or has another meaning. 
3) PRS14458Zamaryam and PRS14459Zamaryam are possibly the same person and whether ቦራራ is a title to them. 
4) I do not know, whether መረዶ and ባጭ፡ አርዋ፤ are place names and where to eventually locate them.

```
<colophon xml:id="coloph1">
                                    <locus target="#159vb"/>
                                    <note>The note states that this manuscript was bought by
                                        <persName role="owner" ref="PRS14454EhataKrestos">ʾƎḫǝta Krǝstos</persName> from
                                        <persName role="bequeather" ref="PRS14455AsqaGiyorgis">ʾAṣqa Giyorgis</persName> and offered to the
                                        <placeName ref="LOC000000000000000000000000000">church of St George</placeName> in the tenth year of the
                                        reign of <persName ref="PRS4015Fasilada">Fāsiladas</persName>, i.e. <date when="1642">1642 AD</date>.</note>
                                    <q xml:lang="gez">ዛቲ፡ ስንክሳር፡ ዘ<persName role="owner" ref="PRS14454EhataKrestos">እኅተ፡ ክርስቶስ፡</persName>
                                        ዘተሣየጠት፡ በ፯ወቄት፡ እም<persName role="bequeather" ref="PRS14455AsqaGiyorgis">አጽቀ፡ ጊዮርጊስ፤</persName>
                                        ወመድኅንሂ፡ <persName ref="PRS14456WalattaDengel"><roleName type="title">ወይዘሮ፡</roleName>ወለተ፡ ድንግል፤</persName>
                                        ወምስክርሂ፡ <persName ref="PRS14457WaldaHawWS">ወልደ፡ ሐዋርያት፡ ወልደ፡ ሰንቡል፤</persName>
                                        ወ<persName ref="PRS14458Zamaryam">ዘማርያም፡</persName> ዘነበረ፡ ወዓሊሁ፤
                                        <persName ref="PRS14459Zamaryam"><roleName type="title">ቦራራ፡</roleName> ዘማርያም፡</persName>
                                        <persName ref="PRS14460Tsaggo"><roleName type="title">አዛዠ፤</roleName> ጸጎ፤</persName>
                                        <persName ref="PRS14461GabraKrestos"><roleName type="title">ቄስ፡ </roleName>ገብረ፡ ክርስቶስ፤</persName>
                                        ወ<persName ref="PRS14462MalkeaKrestos">መልክዓ፡ ክርስቶስ፡</persName>
                                        ዘ<placeName ref="LOC000000000000000000000000">መረዶ፤ </placeName>
                                        <persName ref="PRS14463BahayloZamaryam">በኃይሎ፡ ዘማርያም፡ </persName>
                                        ዘ<placeName ref="LOC0000000000000000000000">እምባጭ፡ አርዋ፤</placeName> (...)</q>
                                </colophon>
```

### CODE REVIEW

File: new/PRS14454EhataKrestos.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">እኅተ፡ ክርስቶስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎḫǝta Krǝstos</persName>
+                    <persName><roleName type="title">wayzaro</roleName></persName>
+                    <floruit notBefore="1640" notAfter="1949">1640ies</floruit>
```
Comment (eu-genia) [inline new/PRS14454EhataKrestos.xml]: notAfter 1649?

File: new/PRS14454EhataKrestos.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                </person>
+                <listRelation>
+                    <relation name="saws:hasOwned" active="PRS14454EhataKrestos" passive="BLorient2082"/>
+                    <relation name="dc:relation" active="PRS14454EhataKrestos" passive="PRS3698ehetaKr"><desc>Possibly the same as 
+                        <persName ref="PRS3698ehetaKr"/>.</desc></relation>
```
Comment (eu-genia) [inline new/PRS14454EhataKrestos.xml:57]: Even if the time fits I doubt this possibility, as the Saint Eheta Krestos in 1642 succeeded Walatta Petros as the abbess of her community; she would not buy a manuscript for a church dedicated to St George (unless they had one on the premises of her monastery of course, but it does not sound right).
 @DenisNosnitsin1970 @sologebre surely know better.

File: new/PRS14454EhataKrestos.xml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            <change who="CH" when="2024-06-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">እኅተ፡ ክርስቶስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎḫǝta Krǝstos</persName>
+                    <persName><roleName type="title">wayzaro</roleName></persName>
```
Comment (eu-genia) [inline new/PRS14454EhataKrestos.xml:50]: Is she really referred to as wayzaro? Could not see this in the quote.
The name sounds sort of monastic to me, whichever the identification.

File: new/PRS14456WalattaDengel.xml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">ወለተ፡ ድንግል፡ </persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walatta Dǝngǝl</persName>
+                    <persName><roleName type="title">wayzaro</roleName></persName>
+                    <floruit notBefore="1640" notAfter="1949">1640ies</floruit>
```
Comment (eu-genia) [inline new/PRS14456WalattaDengel.xml:51]: 1649?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS14454EhataKrestos.xml [hunk 5]
```diff
@@ -48,7 +48,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <persName xml:lang="gez" xml:id="n1">እኅተ፡ ክርስቶስ፡</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎḫǝta Krǝstos</persName>
                     <persName><roleName type="title">wayzaro</roleName></persName>
-                    <floruit notBefore="1640" notAfter="1949">1640ies</floruit>
+                    <death when="1662">1662</death>
+                    <floruit notBefore="1640" notAfter="1962">1640ies to 1662</floruit>
```
Comment (eu-genia) [inline new/PRS14454EhataKrestos.xml:52]: 1662 (6 instead of 9)

File: new/PRS14454EhataKrestos.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS14454EhataKrestos.xml:50]: I assume, it is the same ʾƎḫǝta Krǝstos in the first line and in the bottom part of the colophon.
Screenshot 2024-06-12 110239

**Review by eu-genia — APPROVED** [review]

**Pull request merged**